docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+
+services:
+  db:
+    image: postgres:latest
+    restart: always
+    environment:
+       POSTGRES_USER: ${POSTGRES_USER}
+       POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+       POSTGRES_DB: ${POSTGRES_DB_NAME}
+       POSTGRES_PORT: ${POSTGRES_PORT}
+    ports:
+        - ${POSTGRES_PORT}:${POSTGRES_PORT}
+    volumes:
+        - ./postgres-data:/var/lib/postgresql/data
+  api:
+    build: .
+    command: uwsgi --ini config.ini
+    restart: always
+    volumes:
+      - .:/c3po/
```

Comment:
Just for my own understanding, what do these 2 lines do?

Reply:
This preserves volumes from any previous runs so that data isn't lost. If I change certain code on my laptop, I won't have to rebuild the containers if the correct volume is mounted. It'll keep track of changes and copy over anything that I had in a previous run